src/metabase/cmd/serialization.clj

```diff
@@ -0,0 +1,57 @@
+(ns metabase.cmd.serialization
+  (:require [metabase.db :as mdb]
+            [metabase.models
+             [card :refer [Card]]
+             [collection :refer [Collection]]
+             [dashboard :refer [Dashboard]]
+             [database :refer [Database]]
+             [field :refer [Field] :as field]
+             [metric :refer [Metric]]
+             [pulse :refer [Pulse]]
+             [segment :refer [Segment]]
+             [table :refer [Table]]
+             [user :refer [User]]]
+            [metabase.serialization
+             [dump :as dump]
+             [load :as load]]
+            [schema.core :as s]
+            [toucan.db :as db]
+            [metabase.util
```

Comment:
sorting

Reply:
I'm going to try to write a linter for sorting seems like it would be pretty easy if I could figure out the exact rules cljr-refactor uses